Dockerfile

```diff
@@ -20,17 +23,19 @@ ARG TARGETPLATFORM
 ARG PKG="github.com/distribution/distribution/v3"
 ARG BUILDTAGS="include_oss include_gcs"
 RUN --mount=type=bind,target=/src,rw \
-  --mount=type=cache,target=/root/.cache/go-build \
+  --mount=type=cache,target=/root/.cache \
   --mount=target=/go/pkg/mod,type=cache \
   goreleaser-xx --debug \
     --name="registry" \
     --dist="/out" \
     --main="./cmd/registry" \
     --flags="-v" \
+    --flags="-trimpath" \
     --ldflags="-s -w -X '$PKG/version.Version={{.Version}}' -X '$PKG/version.Revision={{.Commit}}' -X '$PKG/version.Package=$PKG'" \
     --tags="$BUILDTAGS" \
     --files="LICENSE" \
-    --files="README.md"
+    --files="README.md" \
+  && xx-verify --static /usr/local/bin/registry
```

Comment:
Out of curiosity; Do we need both xx and goreleaser-xx ? Where do they overlap, and where are the gaps between the two?

Reply:
Yes there is no static check in goreleaser-xx atm but I could add it or we could switch to xx but would need a dedicated stage to create tarballs which seems fine if we want to.

Reply:
Yeah, mostly it feels like there's a large overlap between the two, so it possible, it would be nice to "pick one" (I'm impartial to "which one", as long as (if possible) it's "one" 😅